README.rst

```diff
@@ -34,3 +34,20 @@ Interactive utility for writing CPython ``Misc/NEWS.d`` entries. See
 the blurb_ directory for more details.
 
 .. _blurb: https://github.com/python/core-workflow/tree/master/blurb
+
+
+Other core workflow tools
+-------------------------
+
++------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------+-------------------------------------------------------+
+| Name                                                                                                 | Description                                                                                | Issue tracker                                                                 | Owner/Maintainer                                      |
++======================================================================================================+============================================================================================+===============================================================================+=======================================================+
+| `python/bedevere <https://github.com/python/bedevere>`__                                             | A bot to help identify missing information for CPython pull requests                       | `Github <https://github.com/python/bedevere/issues>`__                        | `Brett Cannon <https://github.com/brettcannon>`__     |
++------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------+-------------------------------------------------------+
+| `python/the-knights-who-say-ni <https://github.com/python/the-knights-who-say-ni>`__                 | CLA enforcement bot for Python organization projects                                       | `Github <https://github.com/python/the-knights-who-say-ni/issues>`__          | `Brett Cannon <https://github.com/brettcannon>`__     |
```

Comment:
You can also do the following to avoid creating duplicate hyperlink markup:

```rst
`Brett Cannon`_

.. You can now just use `Brett Cannon`_ to link to their GitHub profile in multiple places:

`Brett Cannon`_ is a Python core developer.

.. _`Brett Cannon`: https://github.com/brettcannon
```